specification/storage/Microsoft.BlobStorage/routes.tsp

```diff
@@ -665,8 +650,6 @@ namespace Storage.Blob {
         ...ImmutabilityPolicyExpiresOnResponseHeader;
         ...ImmutabilityPolicyModeResponseHeader;
         ...LegalHoldResponseHeader;
-        ...StructuredBodyTypeResponseHeader;
-        ...StructuredContentLengthResponseHeader;
```

Comment:
@vincenttran-msft why were these headers removed? Does the service actually return these headers?